iOS/Source/BookmarksController.swift

```diff
@@ -117,7 +144,10 @@ extension BookmarksController: UITableViewDelegate {
         }
 
         let editRowAction = UITableViewRowAction(style: .normal, title: "Edit") { (action, indexPath) in
-            // edit and update bookmark
+            guard let path = self.bookmarksDataSource.item(at: indexPath).path else { fatalError("!") }
+            let bookmarkController = BookmarkController(path: path, fetcher: self.fetcher, delegate: self)
```

Comment:
Is this branch outdated? I think I commented that we should send the object here.